### PR TITLE
[Core] Fix delaunator triangle active

### LINKED ISSUE #9280: Problem with delaunator
**Description**

I just updated the master branch and now, I'm not able to compile Kratos. After the first compilation error, I added the DUSE_TETGEN_NONFREE_TPL and DUSE_TRIANGLE_NONFREE_TPL flags, but I still get errors:

```
/home/mmaso/Kratos/kratos/utilities/tessellation_utilities/delaunator_utilities.cpp:20:10: fatal error: delaunator.hpp: No existe el archivo o el directorio
   20 | #include <delaunator.hpp>
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [kratos/CMakeFiles/KratosCore.dir/build.make:1974: kratos/CMakeFiles/KratosCore.dir/utilities/tessellation_utilities/delaunator_utilities.cpp.o] Error 1
make[2]: *** Se espera a que terminen otras tareas....
make[1]: *** [CMakeFiles/Makefile2:424: kratos/CMakeFiles/KratosCore.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

here is the full console log [Documento sin título 1.log](https://github.com/KratosMultiphysics/Kratos/files/7466619/Documento.sin.titulo.1.log) and my configure [configure.sh.txt](https://github.com/KratosMultiphysics/Kratos/files/7466635/configure.sh.txt)
Maybe related to #9231 

**Scope**
- Compilation
- KratosCore
- Documentation

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**
Fix #9280

**🆕 Changelog**
- Update macro (#if instead of #ifdef)
- Missing definition

### CODE REVIEW

**Review by miguelmaso — APPROVED** [review]
thanks

File: applications/MeshingApplication/custom_python/add_meshers_to_python.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@
     #define REAL double
 #endif 
 
-#ifdef USE_TRIANGLE_NONFREE_TPL
+#if USE_TRIANGLE_NONFREE_TPL
```
Comment (philbucher) [inline applications/MeshingApplication/custom_python/add_meshers_to_python.cpp:33]: why doesn't this work?

Comment (loumalouomega) [reply applications/MeshingApplication/custom_python/add_meshers_to_python.cpp:33]: if you define it as false

Comment (philbucher) [reply applications/MeshingApplication/custom_python/add_meshers_to_python.cpp:33]: ah :D

**Pull request merged**